test/unit/math/rev/prob/categorical_logit_glm_lpmf_test.cpp

```diff
@@ -1,10 +1,6 @@
 #include <stan/math/rev.hpp>
-#include <stan/math.hpp>
+#include <stan/math/prim.hpp>
 #include <gtest/gtest.h>
-#include <stan/math/prim/scal/fun/value_of.hpp>
-#include <stan/math/prim/meta/return_type.hpp>
```

Comment:
Shouldn't we keep `stan/math/prim/meta.hpp`?

Reply:
This includes the prim master header, so I thought it doesnt make sense to also include meta.

Reply:
OK, then note that `test/unit/math/rev/meta/operands_and_partials_test.cpp` includes both.

Reply:
Removed that. Thanks!